src/main/java/com/ericsson/ei/waitlist/WaitListWorker.java

```diff
@@ -66,33 +66,43 @@ public TaskScheduler taskScheduler() {
     }
 
     @Scheduled(initialDelayString = "${waitlist.initialDelayResend}", fixedRateString = "${waitlist.fixedRateResend}")
-    public void run() throws JSONException {
-        RulesObject rulesObject;
-        List<String> documents = waitListStorageHandler.getWaitList();
-        for (String document : documents) {
-            JSONObject eventObject = new JSONObject(document);
-            if (eventToObjectMapHandler.isEventInEventObjectMap(eventObject.getString("_id"))) {
-                waitListStorageHandler.dropDocumentFromWaitList(document);
-            } else {
-                String event = eventObject.getString("Event");
-                rulesObject = rulesHandler.getRulesForEvent(event);
-                String idRule = rulesObject.getIdentifyRules();
-
-                if (idRule != null && !idRule.isEmpty()) {
-                    JsonNode ids = jmesPathInterface.runRuleOnEvent(idRule, event);
-                    if (ids.isArray()) {
-                        LOGGER.debug("[EIFFEL EVENT RESENT] id:" + eventObject.getString("_id") + " time:"
-                                + eventObject.getString("Time"));
-                        for (final JsonNode idJsonObj : ids) {
-                            Collection<String> objects = matchIdRulesHandler.fetchObjectsById(rulesObject,
-                                    idJsonObj.textValue());
-                            if (!objects.isEmpty()) {
-                                rmqHandler.publishObjectToWaitlistQueue(event);
-                            }
-                        }
-                    }
-                }
-            }
+    public void run(){
```

Comment:
Why this has been converted to tab when the rest of the file seem to use 4 space indention.